controllers/Endpoint.controller.ts

```
@@ -26,7 +26,7 @@ export class EndpointController {
       logger.debug('Received reveal data via endpoint');
 
       // Extract data from request body
-      const { fundingTx, reveal, l2DepositOwner, l2Sender } = req.body;
+      const { fundingTx, reveal, l2DepositOwner, l2Sender, destinationChainDepositOwner } = req.body;
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)